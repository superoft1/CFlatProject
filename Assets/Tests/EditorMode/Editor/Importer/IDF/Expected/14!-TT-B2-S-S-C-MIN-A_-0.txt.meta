fileFormatVersion: 2
guid: dddd0de95a0ebaf4eb63659697c542a8
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
