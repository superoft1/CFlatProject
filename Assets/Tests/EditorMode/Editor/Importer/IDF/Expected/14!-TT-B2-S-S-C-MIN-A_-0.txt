DiaphramControlValve, FCV-1103, (0.0, -1.0, 0.0), (-124.5, 126.3, 3.9), False
(-124.5, 126.0, 3.9)
(-124.5, 126.5, 3.9)
FreeVertex: 1
WeldNeckFlange, , (0.0, 1.0, 0.0), (-124.5, 126.0, 3.9), False
(-124.5, 126.0, 3.9)
(-124.5, 125.9, 3.9)
FreeVertex: 0
Pipe, , (0.0, -1.0, 0.0), (-124.5, 125.9, 3.9), False
(-124.5, 125.9, 3.9)
(-124.5, 125.9, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-124.5, 125.8, 3.9), False
(-124.5, 125.8, 3.9)
(-124.5, 125.9, 3.9)
FreeVertex: 0
CheckValve, , (0.0, -1.0, 0.0), (-124.5, 125.7, 3.9), True
(-124.5, 125.7, 3.9)
(-124.5, 125.8, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, 1.0, 0.0), (-124.5, 125.6, 3.9), False
(-124.5, 125.7, 3.9)
(-124.5, 125.6, 3.9)
FreeVertex: 0
Pipe, , (0.0, -1.0, 0.0), (-124.5, 125.5, 3.9), False
(-124.5, 125.6, 3.9)
(-124.5, 125.5, 3.9)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-124.5, 125.5, 3.9), False
(-124.5, 125.5, 3.9)
(-124.5, 125.5, 3.9)
(-124.5, 125.5, 3.8)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-124.5, 125.5, 3.7), False
(-124.5, 125.5, 3.8)
(-124.5, 125.5, 3.7)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-124.5, 125.5, 3.7), False
(-124.5, 125.5, 3.6)
(-124.5, 125.5, 3.7)
FreeVertex: 1
Pipe, , (0.0, -1.0, 0.0), (-124.5, 125.4, 3.9), False
(-124.5, 125.5, 3.9)
(-124.5, 125.4, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-124.5, 125.4, 3.9), False
(-124.5, 125.3, 3.9)
(-124.5, 125.4, 3.9)
FreeVertex: 0
Pipe, , (0.0, -1.0, 0.0), (-124.5, 125.3, 3.9), False
(-124.5, 125.3, 3.9)
(-124.5, 125.3, 3.9)
FreeVertex: 0
GateValve, , (0.0, -1.0, 0.0), (-124.5, 125.2, 3.9), False
(-124.5, 125.0, 3.9)
(-124.5, 125.3, 3.9)
FreeVertex: 1
FreeVertexTotal: 3
